### PR TITLE
[RDY] Strain: use Integer.compare

### PR DESCRIPTION
**Pull request by Unia**

### CODE REVIEW

File: dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java [hunk 1]
```diff
@@ -96,13 +96,7 @@ private Range getRange(Bounds bounds) {
...
 	private List<Annotation> getSortedAnnotations(Range ranks) {
 		return graph.getAnnotationByRank(ranks).stream()
-				.sorted((a1, a2) -> {
-					if (a1.getStart() < a2.getStart()) {
-						return -1;
-					} else {
-						return 1;
-					}
-				})
+				.sorted((a1, a2) -> Integer.compare(a1.getStart(), a2.getStart()))
```
Comment (Balletie) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java]: This is not completely the same: `Integer.compare` returns 0 if the two integers are equal, and the lambda expression returns 1 in that case.

Comment (Unia) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java]: Yes, I am aware of that. It should not matter in this case, though.

Comment (Balletie) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java]: I don't think annotations ever have equal start coordinates

**Comment by Balletie** [comment]
Looks fine, next reviewer may merge. :)

**Comment by gfokkema** [comment]
Merging.

**Pull request merged**